core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.scss

```diff
@@ -7,11 +7,7 @@
     padding: $spacing-4;
     padding-bottom: 0;
     height: 100%;
-}
-
-.edit-content-form {
-    overflow: auto;
-    padding-right: $spacing-4;
+    overflow: scroll;
```

Comment:
This will show the scroll even if it's not needed. We should use `overflow: auto;`.